e7-damage-calc/heroes.js

```diff
@@ -1523,14 +1523,15 @@ const heroes = {
     form: [elements.caster_defense, elements.target_hp_pc],
     skills: {
       s1: {
+        soulburn: true,
+        rate: (soulburn) => soulburn ? 0.5 : 0.5,
```

Comment:
```suggestion
        rate: 0.5,
```
no need for the soulburn since it doesn't change the value